README.md

```diff
@@ -93,6 +93,148 @@ File(Code on macOS) > Preferences > Settings to open your [Settings](https://cod
 
 ![ screencast ](images/0.3.0/reg-stack-analysis.gif)
 
+## Using the CRDA Ignore Vulnerability Feature
+
+If a user wishes to ignore all the vulnerabilities, or a specific set of security vulnerabilities of a particular dependency, or set of dependencies, it can be done by adding "crdaignore" followed by an optional list consisting of the security vulnerabilities to ignore as comments in the manifest file for that particular dependency for the maven, pypi and golang ecosystems, or by adding a JSON "crdaignore" object to an npm manifest file as shown in the examples below.
+
+Ignore all the security vulnerabilities present in the flask package in a pypi manifest file.
+
+```
+flask==1.0 #crdaignore
+fastapi==0.36.0 
+sceptre==2.2.1
+```
+ 
+
+Example 2: Ignore a specific set of security vulnerabilities present in the flask package in a pypi manifest file.
+```
+flask==1.0 #crdaignore [vulnerability1, vulnerability2, ..., vulnerability n]
+fastapi==0.36.0 
+sceptre==2.2.1 
+```
+
+Ignore all the security vulnerabilities present in the "ginkgo" and "pax-go" dependencies in a golang manifest file.
+```
+module github.com/fabric8-analytics/acceptance_tests/data
+
+go 1.14
+
+require (
+	code.cloudfoundry.org/archiver v0.0.0-20170223024658-7291196139d7
+	github.com/googleapis/gax-go v1.0.3 crdaignore
+	github.com/googleapis/gax-go/v2 v2.0.5
+	github.com/onsi/ginkgo v1.14.2 // indirect crdaignore
+	github.com/onsi/gomega v1.10.3 // indirect 
+)
+
+```
+Ignore all vulnerabilities of "pax-go", and ignore a specific set of security vulnerabilities present in the "gingko" dependency in a golang manifest file.
+```
+module github.com/fabric8-analytics/acceptance_tests/data
+
+go 1.14
+
+require (
+	code.cloudfoundry.org/archiver v0.0.0-20170223024658-7291196139d7
+	github.com/googleapis/gax-go v1.0.3 crdaignore
+	github.com/googleapis/gax-go/v2 v2.0.5
+	github.com/onsi/ginkgo v1.14.2 // indirect crdaignore [vulnerability 1, vulnerability 2]
+	github.com/onsi/gomega v1.10.3 // indirect 
+)
+```
+Ignore all vulnerabilities of the dependency "junit:junit". 
+```
+<project>
```

Comment:
Just give specific lines in examples, no need to give everything.

    <dependency>
      <groupId>junit</groupId>  <!--crdaignore-->
      <artifactId>junit</artifactId>
      <version>3.8.1</version>
    </dependency>